.github/workflows/go.yml

```diff
@@ -78,6 +78,21 @@ jobs:
     - name: Test registry server with PostgreSQL
       run: go test ./server/registry -postgresql
 
+    - name: Compute code coverage
```

Comment:
I moved this earlier to clarify that it's *not* using the standalone server (notice that the `APG_REGISTRY` environment variable settings are also removed)

Reply:
Is there any reason to not to combine this with the `Test everything with SQLite` test?

Reply:
Hmm. If we have postgres-specific code (where is that?) and that is our targeted production platform, then I think we should emit the coverage with the postgres test.